composer.json

```diff
@@ -23,6 +23,7 @@
         "ezyang/htmlpurifier": "^4.10",
         "geoip2/geoip2": "~2.4.2",
         "greenlion/php-sql-parser": "^4.1",
+        "guzzlehttp/cache-subscriber": "dev-master",
```

Comment:
What is the reason to add this dependency?

Reply:
Error :stuck_out_tongue: